core/templates/components/ck-editor-helpers/ck-editor-4-widgets.initializer.ts

```diff
@@ -152,10 +152,12 @@ export class CkEditorInitializerService {
                     if (widgetShouldBeRemoved) {
                       var newWidgetSelector = (
                         '[data-cke-widget-id="' + that.id + '"]');
-                      var widgetElement = editor.editable().findOne(
-                        newWidgetSelector);
-                      if (widgetElement) {
-                        widgetElement.remove();
+                      if (newWidgetSelector !== null) {
```

Comment:
Was this a temporary issue? As is it is onyl null when the page is loading and gets its value later?
Since otherwise this condition may never become true, right?

Reply:
Thanks @aks681. Actually, I wasn't able to reproduce this error on the localhost with and without the `--prod_env flag`. In my case, the concept card modal opens instantaneously providing no opportunity to exit the New Skill editor first. It probably assumes `null` in the case when the original editor is exited first.
So, I added a `null` check to prevent the console errors.